SMLHelper/ThunderstoreMetadata/BZ.EXP/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "SMLHelper_BZ_Experimental",
-  "version_number": "2.15.0001",
+  "version_number": "3.0.0",
   "website_url": "https://github.com/SubnauticaModding/SMLHelper/wiki",
   "description": "SMLHelper is a modding library that helps making mods easier by helping with adding new items, changing items, adding models, sprites, etc.",
   "dependencies": [ "Subnautica_Modding-BepInExPack-5.4.2101" ]
```

Comment:
1. Are we even going to support thunderstore anymore?
2. if we are then I propose NOT having the exp branches on them anymore either way.